scripts/docker/dev/start-develop.sh

```diff
@@ -15,6 +15,17 @@ if [ "${TEST}" != "1" ]; then
     make migrate
 fi
 
+# clone or pull the repo: galaxy-lint-rules
```

Comment:
1. Moved clone to dev/Dockerfile and release/Dockerfile.build
2. Cloned to root rather than project source tree. Moved pull to command: make dev/lint-rule-update